docs/source/conf.py

```diff
@@ -461,6 +461,7 @@ def get_tokens_unprocessed(self, text):
     'cryptography.x509.extensions.Extension': 'cg:cryptography.x509.Extension',
     'ExtensionTypeVar': 'cg:cryptography.x509.ExtensionType',
     'Union': 'python:typing.Union',
+    'typing.S': 'django_ca.extensions.base.S',
```

Comment:
That fixed the weird error I reported about at https://github.com/mathiasertl/django-ca/pull/69#issuecomment-758883966.
```
<unknown>:1:py:obj reference target not found: typing.S
```